### ISSUE TITLE
Push Docker image to GHCR

### ISSUE THREAD

**Issue by onedr0p** [JasonHHouse/gaps#259]
**Is your feature request related to a problem? Please describe.**

Dockerhub ratelimits image pulls and is generally considered not the best place to host docker images anymore.

**Describe the solution you'd like**

In the github workflows, push to the Github Container Registry (GHCR)

**Comment by JasonHHouse** [comment]
Makes sense. I won't remove Dockerhub as that's used by many but I can look into adding GHCR as another option.
